.github/workflows/build-container.yml

```diff
@@ -0,0 +1,35 @@
+name: build-docker-image
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '7 0 * * sun'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+
```

Comment:
```suggestion
```

I think this is only necessary if you want to pass variables between additional jobs

Reply:
okie doke.